packages/web-components/@sfx/ui/stories/index.ts

```diff
@@ -0,0 +1,27 @@
+import { storiesOf } from '@storybook/html';
+import { withKnobs, text } from '@storybook/addon-knobs';
+import '../src/index.ts';
```

Comment:
The extension is not necessary (I'm surprised TypeScript didn't yell at you for this).